src/components/molecules/Forms/CreateClaim/CreateClaim.scss

```diff
@@ -0,0 +1,40 @@
+@import 'colors';
+
+.CreateClaim {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 100%;
+
+  &__name {
+    width: 49%;
+    margin-right: .5rem;
+  }
+
+  &__author {
+    width: 49%;
+  }
+
+  &__datePublished {
+    width: 32.66%;
+    margin-right: .25rem;
+  }
+
+  &__dateCreated {
+    width: 32.66%;
+    margin-right: .25rem;
+  }
+
+  &__tags {
+    width: 32.66%;
+  }
+
+  &__content {
+    height: 4rem;
+    /* stylelint-disable */
+    width: calc(98% + .5rem);
```

Comment:
is possible to avoid calc here?

Reply:
Ok I updated it so that I didn't have to use calc to keep the boxes the same size.